allure-attachments/src/main/java/io/qameta/allure/attachment/http/HttpRequestAttachment.java

```diff
@@ -41,16 +41,19 @@ public class HttpRequestAttachment implements AttachmentData {
 
     private final Map<String, String> cookies;
 
+    private final Map<String, String> formParams;
+
     public HttpRequestAttachment(final String name, final String url, final String method,
                                  final String body, final String curl, final Map<String, String> headers,
-                                 final Map<String, String> cookies) {
+                                 final Map<String, String> cookies, final Map<String, String> formParams) {
```

Comment:
Let's keep the old constructor and create a new one to make the change backwards-compatible.

Reply:
fixed